lightly/embedding/embedding.py

```diff
@@ -123,16 +133,26 @@ def embed(self,
 
                 process_time = time.time()
 
-                efficiency = \
-                    (process_time - prepare_time) / (process_time - start_time)
-                pbar.set_description(
-                    "Compute efficiency: {:.2f}".format(efficiency))
+                efficiency = (process_time - prepare_time) / (process_time - start_time)
+                pbar.set_description("Compute efficiency: {:.2f}".format(efficiency))
                 start_time = time.time()
 
             embeddings = torch.cat(embeddings, 0)
             labels = torch.cat(labels, 0)
-            if to_numpy:
-                embeddings = embeddings.cpu().numpy()
-                labels = labels.cpu().numpy()
 
-        return embeddings, labels, fnames
+            embeddings = embeddings.cpu().numpy()
+            labels = labels.cpu().numpy()
+
+        list_to_order = zip(embeddings, labels)
+        dict_by_filenames = dict(zip(fnames, list_to_order))
```

Comment:
the [style guide](https://google.github.io/styleguide/pyguide.html#3161-names-to-avoid) recommends to avoid names which include the type of the variable.